### PR TITLE
Add RunStmt structure and placeholder implementation for ExecuteRun method.

### PR DESCRIPTION
**Pull request by brightcoder01**
Relate #2161

### CODE REVIEW

File: pkg/ir/ir.go [hunk 1]
```diff
@@ -298,3 +299,29 @@ func (sql *OptimizeStmt) IsExtended() bool { return true }
...
+	// ImageName is the name of the docker image after TO RUN keyword.
+	ImageName string
+	// Parameters is the command line parameters for the docker image.
+	Parameters []string
+	// Into is the output table names (0~N, comma separated) after INTO keyword.
+	Into string
+}
+
+// SetOriginalSQL sets the original sql string
+func (cl *RunStmt) SetOriginalSQL(s string) { cl.OriginalSQL = s }
```
Comment (weiguoz) [inline pkg/ir/ir.go:318]: What's `cl` stands for?

File: pkg/ir/ir_generator_test.go [hunk 2]
```diff
@@ -267,3 +268,110 @@ func TestGenerateTrainStmtModelZoo(t *testing.T) {
...
+		a.Equal(`a_data_scientist/ts_data_processor:1.0`, runStmt.ImageName)
+		a.True(reflect.DeepEqual(runStmt.Parameters, []string{`slide_window_to_row`}))
+		a.Equal(`output_table_1,output_table_2`, runStmt.Into)
+	}
+
+	{
+		testToRun := `
+SELECT * FROM source_table 
+TO RUN a_data_scientist/ts_data_processor:1.0
+CMD "slide_window_to_row", "--param_a=value_a", "--param_b=value_b"
```
Comment (weiguoz) [inline pkg/ir/ir_generator_test.go:356]: I wonder if we support parsing `CMD "slide_window_to_row", "--param_a=va,vb", "--param_b=value_b"`.
Shall we add such a test?

**Review by weiguoz — APPROVED** [review]
LGTM
Let's keep moving.

**Review by sneaxiy — APPROVED** [review]
LGTM++.

File: pkg/ir/ir.go (hunk 1 above)
Comment (brightcoder01) [reply pkg/ir/ir.go:318]: It stands for `clause`.

File: pkg/ir/ir_generator_test.go (hunk 2 above)
Comment (brightcoder01) [reply pkg/ir/ir_generator_test.go:356]: Yes, it can parse.
Will add the test case in the following PR.

**Pull request merged**